k8s/object.bzl

```diff
@@ -515,42 +553,41 @@ def k8s_object(name, **kwargs):
     _k8s_object(name = name, **resolve_args)
     _k8s_object(name = name + ".resolve", **resolve_args)
 
-    if "cluster" in kwargs or "context" in kwargs:
-        _k8s_object_create(
-            name = name + ".create",
-            resolved = name,
-            **common_args
-        )
-        _k8s_object_delete(
-            name = name + ".delete",
-            resolved = name,
-            **common_args
-        )
-        _k8s_object_replace(
-            name = name + ".replace",
-            resolved = name,
-            **common_args
-        )
-        _k8s_object_apply(
-            name = name + ".apply",
-            resolved = name,
+    # Need to rework this logic to enable the check again
+    # if kwargs["clusters"][ClusterAspectInfo].cluster != None or kwargs["clusters"][ClusterAspectInfo].context != None:
+    _k8s_object_create(
+        name = name + ".create",
+        resolved = name,
+        **common_args
+    )
+    _k8s_object_delete(
+        name = name + ".delete",
+        resolved = name,
+        **common_args
+    )
+    _k8s_object_replace(
+        name = name + ".replace",
+        resolved = name,
+        **common_args
+    )
+    _k8s_object_apply(
+        name = name + ".apply",
+        resolved = name,
+        **common_args
+    )
+    _k8s_object_diff(
+        name = name + ".diff",
+        resolved = name,
+        kind = kwargs.get("kind"),
+        clusters = kwargs.get("clusters"),
+        user = kwargs.get("user"),
+        namespace = kwargs.get("namespace"),
```

Comment:
```suggestion
        namespaces = kwargs.get("namespaces"),
        namespace = kwargs.get("namespace"),
```

Reply:
I will hide this comment. This pr moves cluster object but internal patch also does some namespace stuff.